### PR TITLE
Small improvements to /calc-send

### PR DESCRIPTION
**Pull request by clifffrey**
Specifically, this fixes an issue if player_count is set to something that does not parse as a number.  Additionally, we now accept a few more spellings of things.

### Tested Changes:
- [X] I've tested the changes locally.
- [ ] I've not tested the changes.

### CODE REVIEW

File: maps/biter_battles_v2/feeding_calculations.lua [hunk 1]
```diff
@@ -124,18 +124,18 @@ function Public.calc_send_command(params, difficulty_vote_value, bb_evolution, m
 					error_msg = "Invalid difficulty"
 				end
 			elseif k == "players" then
-				player_count = math_floor(tonumber(v))
-				if player_count == nil or player_count < 0 or player_count > 10000 then
+				player_count = tonumber(v)
+				if player_count == nil or player_count < 0 or player_count > 10000 or player_count ~= math.floor(player_count) then
```
Comment (XVhc6A) [inline maps/biter_battles_v2/feeding_calculations.lua:128]: makes sense! math.floor moved.  You may want to use `math_floor` as it is defined at the top.  The rest of the file inconsistently uses `math.floor` and `math_floor`, so maybe work it to unify that.   Not a blocker to this PR though.

File: maps/biter_battles_v2/feeding_calculations.lua [hunk 2]
```diff
@@ -124,18 +124,18 @@ function Public.calc_send_command(params, difficulty_vote_value, bb_evolution, m
...
-				if v == "green" then v = "logistic-science-pack" end
-				if v == "gray" or v == "grey" then v = "military-science-pack" end
-				if v == "blue" then v = "chemical-science-pack" end
-				if v == "purple" then v = "production-science-pack" end
-				if v == "yellow" then v = "utility-science-pack" end
-				if v == "white" then v = "space-science-pack" end
+			elseif k == "color" or k == "colour" then
+				if v == "red" or v == "automation" then v = "automation-science-pack" end
+				if v == "green" or v == "logistic" then v = "logistic-science-pack" end
+				if v == "gray" or v == "grey" or v == "black" or v == "military" then v = "military-science-pack" end
```
Comment (XVhc6A) [inline maps/biter_battles_v2/feeding_calculations.lua:134]: nice, I alwaays called it black sci too.

**Review by XVhc6A — APPROVED** [review]
looks good, overall minor changes that will positively affect QOL

File: maps/biter_battles_v2/feeding_calculations.lua (hunk 1 above)
Comment (clifffrey) [reply maps/biter_battles_v2/feeding_calculations.lua]: done, using `math_floor` consistently now.  Thanks!

**Pull request merged**